src/athesis/operators.jl

```diff
@@ -6,81 +6,81 @@
 # Upwind-y
-@inline δfδy_3dᵘ(f,i,j,k,Δy)   = (f[i,j,k] - f[i,j-1,k])/Δy
+@inline δfδy_3dᵘ(f, i, j, k, Δy) = (f[i,j,k] - f[i,j - 1,k]) / Δy
 # Downwind-y
-@inline δfδy_3dᵈ(f,i,j,k,Δy) = (f[i,j+1,k] - f[i,j,k])/Δy
+@inline δfδy_3dᵈ(f, i, j, k, Δy) = (f[i,j + 1,k] - f[i,j,k]) / Δy
 # Central-z
-@inline δfδz_3dᶜ(f,i,j,k,Δz)  = (f[i,j,k+1] - f[i,j,k-1])/(2Δz)
+@inline δfδz_3dᶜ(f, i, j, k, Δz) = (f[i,j,k + 1] - f[i,j,k - 1]) / (2Δz)
 # Upwind-z
-@inline δfδz_3dᵘ(f,i,j,k,Δz)   = (f[i,j,k] - f[i,j,k-1])/Δz
+@inline δfδz_3dᵘ(f, i, j, k, Δz) = (f[i,j,k] - f[i,j,k - 1]) / Δz
 # Downwind-z
-@inline δfδz_3dᵈ(f,i,j,k,Δz) = (f[i,j,k+1] - f[i,j,k])/Δz
+@inline δfδz_3dᵈ(f, i, j, k, Δz) = (f[i,j,k + 1] - f[i,j,k]) / Δz
 
 ############### 2d gradient operators ##################
 # central gradient
-@inline ∇2dᶜ(f,i,j,(Δx,Δy)) = (δfδx_2dᶜ(f,i,j,Δx), δfδy_2dᶜ(f,i,j,Δy))
+@inline ∇2dᶜ(f, i, j, (Δx, Δy)) = (δfδx_2dᶜ(f, i, j, Δx), δfδy_2dᶜ(f, i, j, Δy))
 # upwind gradient
-@inline ∇2dᵘ(f,i,j,(Δx,Δy)) = (δfδx_2dᵘ(f,i,j,Δx), δfδy_2dᵘ(f,i,j,Δy))
+@inline ∇2dᵘ(f, i, j, (Δx, Δy)) = (δfδx_2dᵘ(f, i, j, Δx), δfδy_2dᵘ(f, i, j, Δy))
 # downwind gradient
-@inline ∇2dᵈ(f,i,j,(Δx,Δy)) = (δfδx_2dᵈ(f,i,j,Δx), δfδy_2dᵈ(f,i,j,Δy))
+@inline ∇2dᵈ(f, i, j, (Δx, Δy)) = (δfδx_2dᵈ(f, i, j, Δx), δfδy_2dᵈ(f, i, j, Δy))
 
 ############### 3d gradient operators ##################
 # central gradient
-@inline ∇3dᶜ(f,i,j,k,(Δx,Δy,Δz)) = (δfδx_3dᶜ(f,i,j,k,Δx), δfδy_3dᶜ(f,i,j,k,Δy), δfδz_3dᶜ(f,i,j,k,Δz))
+@inline ∇3dᶜ(f, i, j, k, (Δx, Δy, Δz)) = (δfδx_3dᶜ(f, i, j, k, Δx), δfδy_3dᶜ(f, i, j, k, Δy), δfδz_3dᶜ(f, i, j, k, Δz))
 # upwind gradient
-@inline ∇3dᵘ(f,i,j,k,(Δx,Δy,Δz)) = (δfδx_3dᵘ(f,i,j,k,Δx), δfδy_3dᵘ(f,i,j,k,Δy), δfδz_3dᵘ(f,i,j,k,Δz))
+@inline ∇3dᵘ(f, i, j, k, (Δx, Δy, Δz)) = (δfδx_3dᵘ(f, i, j, k, Δx), δfδy_3dᵘ(f, i, j, k, Δy), δfδz_3dᵘ(f, i, j, k, Δz))
 # downwind gradient
-@inline ∇3dᵈ(f,i,j,k,(Δx,Δy,Δz)) = (δfδx_3dᵈ(f,i,j,k,Δx), δfδy_3dᵈ(f,i,j,k,Δy), δfδz_3dᵈ(f,i,j,k,Δz))
+@inline ∇3dᵈ(f, i, j, k, (Δx, Δy, Δz)) = (δfδx_3dᵈ(f, i, j, k, Δx), δfδy_3dᵈ(f, i, j, k, Δy), δfδz_3dᵈ(f, i, j, k, Δz))
 
 ############### 2d divergence operators ##################
 # central divergence
-@inline div2dᶜ(f,i,j,(Δx,Δy)) = δfδx_2dᶜ(f,i,j,Δx) + δfδy_2dᶜ(f,i,j,Δy)
+@inline div2dᶜ(f, i, j, (Δx, Δy)) = δfδx_2dᶜ(f, i, j, Δx) + δfδy_2dᶜ(f, i, j, Δy)
 # upwind divergence
-@inline div2dᵘ(f,i,j,(Δx,Δy)) = δfδx_2dᵘ(f,i,j,Δx) + δfδy_2dᵘ(f,i,j,Δy)
+@inline div2dᵘ(f, i, j, (Δx, Δy)) = δfδx_2dᵘ(f, i, j, Δx) + δfδy_2dᵘ(f, i, j, Δy)
 # downwind divergence
-@inline div2dᵈ(f,i,j,(Δx,Δy)) = δfδx_2dᵈ(f,i,j,Δx) + δfδy_2dᵈ(f,i,j,Δy)
+@inline div2dᵈ(f, i, j, (Δx, Δy)) = δfδx_2dᵈ(f, i, j, Δx) + δfδy_2dᵈ(f, i, j, Δy)
 
 ############### 3d divergence operators ##################
 # central divergence
-@inline div3dᶜ(f,i,j,k,(Δx,Δy,Δz)) = (δfδx_3dᶜ(f,i,j,k,Δx) + δfδy_3dᶜ(f,i,j,k,Δy) + δfδz_3dᶜ(f,i,j,k,Δz))
+@inline div3dᶜ(f, i, j, k, (Δx, Δy, Δz)) = (δfδx_3dᶜ(f, i, j, k, Δx) + δfδy_3dᶜ(f, i, j, k, Δy) + δfδz_3dᶜ(f, i, j, k, Δz))

```

Comment:
Maybe it is not possible, but here we might want to have the option to leave out the whitespace for compactness